content/docs/command-reference/remote/modify.md

```diff
@@ -310,19 +310,99 @@ $ dvc remote modify myremote endpointurl \
   $ dvc remote modify myremote url azure://mycontainer/path
   ```
 
-- `connection_string` - connection string:
+- `connection_string` -
+  [connection string](http://azure.microsoft.com/en-us/documentation/articles/storage-configure-connection-string/):
 
   ```dvc
   $ dvc remote modify --local myremote connection_string \
                               'mystring'
   ```
 
-> The connection string contains sensitive user info. Therefore, it's safer to
-> add it with the `--local` option, so it's written to a Git-ignored config
-> file.
+  > The connection string contains sensitive user info. Therefore, it's safer to
+  > add it with the `--local` option, so it's written to a Git-ignored config
+  > file.
 
-For more information on configuring Azure Storage connection strings, visit
-[here](https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string).
+  For more information on configuring Azure Storage connection strings, visit
+  [here](https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string).
+
+* `account_name` - storage account name, will be required if authenticated
+  either with `account_key` or `sas_token`.
+
+  ```dvc
+  $ dvc remote modify --local myremote account_name \
+                              'account_name'
+  ```
+
+* `account_key` - storage account key:
+
+  ```dvc
+  $ dvc remote modify --local myremote account_key \
+                              'account_key'
+  ```
+
+* `sas_token` - shared access signature token, can be used instead of
+  `account_key`:
+
+  ```dvc
+  $ dvc remote modify --local myremote sas_token \
+                              'sas_token'
+  ```
+
+* `tenant_id` - Tenant ID for
```

Comment:
unless is is a proper name, all options start with a small letter - Tenant -> tenant